portafly/src/App.tsx

```diff
@@ -17,60 +19,52 @@ const navItems = [
     title: 'Analytics',
     to: '/analytics',
     items: [
-      {to: '/analytics/usage', title: 'Usage'},
+      { to: '/analytics/usage', title: 'Usage' },
     ],
   },
   {
     title: 'Applications',
     to: '/applications',
     items: [
-      {to: '/applications', title: 'Listing'},
-      {to: '/applications/plans', title: 'Application Plans'}
+      { to: '/applications', title: 'Listing' },
+      { to: '/applications/plans', title: 'Application Plans' },
     ],
   },
   {
     title: 'Integration',
     to: '/integration',
     items: [
-      {to: '/integration/configuration', title: 'Configuration'}
-    ]
-  }
+      { to: '/integration/configuration', title: 'Configuration' },
+    ],
+  },
```

Comment:
Last items in a list (array, obj, etc) should not enforce trailing commas.

Reply:
ok